linux/debian/8/foxpass_setup.py

```diff
@@ -49,9 +51,9 @@ def main():
 
     apt_get_update()
     install_dependencies()
-    write_foxpass_ssh_keys_script(args.api_url, args.api_key)
-    write_nslcd_conf(uri=args.ldap_uri, basedn=args.base_dn, binddn=binddn, bindpw=args.bind_pw,
-                     threads=int(args.ldap_connections))
+    write_foxpass_ssh_keys_script(args.api_url, args.apis, args.api_key)
+    write_nslcd_conf(uri=args.ldap_uri, ldaps=args.ldaps, basedn=args.base_dn,
```

Comment:
combine args.ldap_uri and args.ldaps into one array to send into write_nslcd_conf()